core/src/main/java/com/linecorp/armeria/client/endpoint/OrElseEndpointGroup.java

```diff
@@ -42,4 +42,10 @@
         }
         return second.endpoints();
     }
+
+    @Override
+    public void close() {
+        first.close();
+        second.close();
```

Comment:
Probably better calling `second.close()` in `finally {}`?

Reply:
Used try / resources